src/clear/migration/manager.cr

```diff
@@ -137,9 +137,12 @@ class Clear::Migration::Manager
     ensure_ready
 
     list_of_migrations = @migrations.sort { |a, b| a.uid <=> b.uid }
-    list_of_migrations.reject! { |x| @migrations_up.includes?(x) }
+    list_of_migrations.reject! { |x| @migrations_up.includes?(x.uid) }
```

Comment:
Heart of the problem - @migrations_up is a list of the uids already applied, not the migrations themselves. Need to compare apples to apples :)